homeassistant/components/modbus/manifest.json

```diff
@@ -4,5 +4,6 @@
   "documentation": "https://www.home-assistant.io/integrations/modbus",
   "requirements": ["pymodbus==2.5.1"],
   "codeowners": ["@adamchengtkc", "@janiversen", "@vzahradnik"],
+  "version": "2021.5",
```

Comment:
This shouldn't be here. Core components are built in, adding this will only cause duplicate data.

Reply:
But the whole purpose of this PR are to make the HA integration manifest identical to custom_components, so that users can easy install old versions as custom_components.

If not we need to tell the user to copy the component and then also modify the manifest.json

Reply:
I agree it adds "version" but that is not as such a duplication of data, it is merely not used in HA integrations.

Reply:
It is loaded (while not needed), it causes also other problems; like: when do we update this? Merge conflicts for patch releases during cherry picking and all sorts of things.

We allow custom integrations, but we don't support them and most certainly should not pollute the core for it.